README.md

```diff
@@ -9,18 +11,21 @@ An example Next.js website that shows Postgres full text search, next-image with
 1. Create a new Supabase project, or use an existing one
 2. Run the `schema.sql` file in the Supabase dashboard to create the relevant tables
 3. Update the environment variables (either in Vercel or locally) with Supabase URL and Anon Key
-4. Deploy Supabase function for contract form notification:
+
+- Locally: `cp .env.local.example .env.local`
+
+4. Deploy Supabase Edge Function for contact form notifications:
 
 ```
 supabase link --project-ref your-project-ref
-supabase functions deploy contact-notification
 supabase secrets set SMTP_HOSTNAME="your.hostname.com" SMTP_PORT="2587" SMTP_USERNAME="your_username" SMTP_PASSWORD="your_password" SMTP_FROM="no-reply@example.com" SMTP_TO="you@example.com" FUNCTION_SECRET="your-random-secret"
+supabase functions deploy contact-notification
```

Comment:
I had it this way round because I know it works - are you allowed to set secrets before a function is deployed? If you are, this is much cleaner!

Reply:
Yes, secrets are set per project, so as long as you've done the `link` step you can set secrets. I actually thought you have to set them before deploying for them to be available in your functions, but not sure.
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/5748289/163945652-1d3373a4-ef24-4184-8ce6-2d26b3de79e4.png">

Reply:
Great! I think whenever you set secrets, it even redeploys behind the scenes, so both ways would work